### PR TITLE
Improving readability of argparse.rst

### PR DESCRIPTION
**Pull request by xandrade**
Fixed few typos and enhanced few paragraphs for your review and consideration.
Trivial contribution towards continuous improvement, so no issue was raised.

Automerge-Triggered-By: GH:rhettinger

### CODE REVIEW

File: Doc/library/argparse.rst [hunk 1]
```diff
@@ -119,7 +119,7 @@ the command-line integers:
    $ python prog.py 1 2 3 4 --sum
    10
 
-If invalid arguments are passed in, it will issue an error:
```
Comment (Mariatta) [inline Doc/library/argparse.rst:122]: I would keep this unchanged.

File: Doc/library/argparse.rst [hunk 2]
```diff
@@ -91,8 +91,8 @@ produces either the sum or the max::
    args = parser.parse_args()
    print(args.accumulate(args.integers))
 
-Assuming the Python code above is saved into a file called ``prog.py``, it can
-be run at the command line and provides useful help messages:
```
Comment (Mariatta) [inline Doc/library/argparse.rst:95]: I would keep the original text on both lines here.

File: Doc/library/argparse.rst [hunk 3]
```diff
@@ -22,8 +22,8 @@
 The :mod:`argparse` module makes it easy to write user-friendly command-line
 interfaces. The program defines what arguments it requires, and :mod:`argparse`
 will figure out how to parse those out of :data:`sys.argv`.  The :mod:`argparse`
-module also automatically generates help and usage messages and issues errors
-when users give the program invalid arguments.
+module. Moreover, it automatically generates the "help and usage messages" and 
+"issues errors" when users provide invalid arguments to the program.
```
Comment (Mariatta) [inline Doc/library/argparse.rst]: Please keep the original text on this pargraph.

**Review by Mariatta — CHANGES_REQUESTED** [review]
Some of the changes here make sense, but the rest, I don't think the change adds any more value, so I'd like to keep them.

I've identified the ones which I'd like you to revert.

**Review by rhettinger — APPROVED** [review]

**Pull request merged**